### DISCUSSION TITLE
TS2339: Property 'social' does not exist on type '{ pageId: "login-without-wallets.ftl"; } & Common'.

### DISCUSSION THREAD

**Discussion by ghost** [keycloakify/keycloakify#368] [Q&A]
I made a copy of login.tsx file with the name LoginWithOutWallets.tsx and want to use the same code of login.tsx but with some changes and i encountered the error TS2339: Property 'social' does not exist on type '{ pageId: "login-without-wallets.ftl"; } & Common'. how do i solve it?

**Comment by garronej** [comment]
Hi @kamranhafeezbim,  

Do not rename the page just keep `login.ftl`. Keycloak don't know how to root to your page.
